Source/WTF/wtf/SortedArrayMap.h

```diff
@@ -161,7 +162,7 @@ template<ASCIISubset subset, typename CharacterType> constexpr std::make_unsigne
 template<ASCIISubset subset> constexpr ComparableASCIISubsetLiteral<subset>::ComparableASCIISubsetLiteral(ASCIILiteral inputLiteral)
     : literal { inputLiteral }
 {
-    ASSERT_UNDER_CONSTEXPR_CONTEXT(std::all_of(literal.span().begin(), literal.span().end(), [] (char character) {
+    ASSERT_UNDER_CONSTEXPR_CONTEXT(std::ranges::all_of(literal.span(), [] (char character) {
```

Comment:
While changing this line, want to fix the syntax to remove the whitespace between the [] and the (char characters)?

(https://webkit.org/code-style-guidelines/#spacing-lambda-paren)

Reply:
Sure.